sns/ut/parity_math_ut.c

```diff
@@ -800,13 +804,36 @@ static void rhs_prepare(const struct m0_sns_ir *ir, struct m0_matvec *des,
 static void reconstruct(const struct m0_sns_ir *ir, const struct m0_matvec *b,
 			struct m0_matvec *r)
 {
+#if ISAL_ENCODE_ENABLED
```

Comment:
We check "#if ISAL_ENCODE_ENABLED" here.
But we check "#if RS_ENCODE_ENABLED" in next function.
I'd prefer to check the __SAME__ macro in all places.
This will help improve code readability.

Reply:
Can the "ISAL_ENODE_ENABLED" and "RS_ENCODE_ENABLED" be true at the same time?
If not, it is OK to check one of them.
If yes, we must make sure only one of them is true.

So, it is better to check only one of them, and assert the other one is not defined.

Reply:
No. `ISAL_ENODE_ENABLED` and `RS_ENCODE_ENABLED` can not be true at the same time. In ana/parity_defs.h 
`#define RS_ENCODE_ENABLED	(!ISAL_ENCODE_ENABLED)`.
Some code in parity_math.c is specific to RS so I had added `RS_ENCODE_ENABLED`.

So shall I remove `RS_ENCODE_ENABLED` from all files and check only `ISAL_ENODE_ENABLED`?

Reply:
I think we only need defining "ISAL_ENCODE_ENABLED".
Let's remove RS_ENCODE_ENABLED.

Reply:
Done